llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -2066,6 +2066,24 @@ def int_amdgcn_interp_inreg_p2_f16 :
             [IntrNoMem, IntrSpeculatable,
              ImmArg<ArgIndex<3>>]>;
 
+// llvm.amdgcn.interp.inreg.p10.rtz.f16 <p>, <i>, <p0>, <high>
+// fp16 interpolation intrinsic, but round-toward-zero.
+// high selects whether high or low 16-bits are used for p and p0 operands
+def int_amdgcn_interp_inreg_p10_rtz_f16:
```

Comment:
inreg should go to the end, or gain a gfx11 suffix or something. Shouldn't split the real opcode name up

Reply:
Maybe we can drop the inreg suffix for the rtz version intrinsic name, there is no precedence of such rtz version, so we don't really need to have such suffix exposed. Any comment?

Reply:
I think that sounds OK. @piotrAMD any thoughts?

Reply:
Yeah, dropping inreg from the rtz versions sounds good, but please add a comment in llvm/include/llvm/IR/IntrinsicsAMDGPU.td that the new intrinsics you are adding are GFX11+, to avoid confusion.

Reply:
Yeah, make sense.